src/alert_rules/prometheus/prometheus_alerts.yaml

```diff
@@ -11,16 +11,26 @@
     "labels":
       "severity": "critical"
 
-  - "alert": "OpenSearchClusterYellow"
+  - "alert": "OpenSearchClusterYellowTemp"
     "annotations":
-      "message": "Cluster {{ $labels.cluster }} health status has been YELLOW for at least 20m. Some cluster replicas shards are not allocated."
-      "summary": "Cluster health status is YELLOW"
+      "message": "Cluster {{ $labels.cluster }} health status has been YELLOW for at least 20m. Cluster might be under heavy load."
+      "summary": "Cluster health status is temporarily YELLOW"
     "expr": |
-      sum by (cluster) (opensearch_cluster_status == 1)
+      sum by (cluster, instance) (opensearch_cluster_shards_number{type=~"relocating|initializing"}) > 0 and on(cluster, instance) opensearch_cluster_status == 1
```

Comment:
@gabrielcocenza breaking per instance here?

Reply:
*note* The timing seems perfect: I agree we should wait for a longer time before raising the "YELLOW_TEMP" vs. "YELLOW", as yellow is a more serious condition and yellow temp may just be consequence of a busy cluster,

Reply:
I thought about breaking per instance to try to show which "node" it's not able to host the shard. In a theoretic scenario nodes can have different capacity.

However I think it would be ok just break for cluster. What do you think?

Reply:
makes sense, but it will be X alerts at once, or very closely I guess?

Reply:
We should not have an alert at the instance level. We should alerts on the cluster level and with a full view over the instances.